Bottom: 0ff4a98f70bbcdd4755bb46c5cdbad657698a8cf
Top:    40a83ebe74d2965358e614a8e204b9402254c9e0
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:21:01 -0500

Add new host buildbot-worker.novatech-llc.com

Runs buildbot worker containers


---
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index e544fd42..6c319766 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -3,6 +3,10 @@ testdaedalus.novatech-llc.com
 
 [docker-hosts]
 testdaedalus.novatech-llc.com
+buildbot-worker.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[buildbot-worker]
+buildbot-worker.novatech-llc.com
